### DISCUSSION TITLE
Hacker News!

### DISCUSSION THREAD

**Discussion by leejustin** [xyflow/xyflow#1119] [General]
Congrats @moklick and team for the traction on the Hacker News post 😄  I have people in my professional network who started talking about React Flow and looking into it as a possible solution for some of their UI problems because of this post!

**Comment by moklick** [comment]
Thanks :) I hope that I find more time to work on some of the issues and new features. Especially grouping and sub flows would be really nice to have I think. And thanks for your help answering some issues :) that's great!
